code/modules/mob/living/simple_animal/familiars/familiars.dm

```diff
@@ -189,10 +155,44 @@
 	holder_type = /obj/item/weapon/holder/cat
 	mob_size = MOB_SMALL
 
-	health = 25
-	maxHealth = 25
+	health = 45
+	maxHealth = 45
 	melee_damage_lower = 3
 	melee_damage_upper = 4
 	attacktext = "clawed"
+	density = 0
 
 	wizardy_spells = list(/spell/targeted/subjugation)
+
+
+/mob/living/simple_animal/mouse/familiar
+	name = "ancient mouse"
+	desc = "A small rodent. It looks very old."
+	body_color = "gray"
+
+
+	see_in_dark = 8
+	see_invisible = SEE_INVISIBLE_NOLIGHTING
+
+
+	health = 25
+	maxHealth = 25
+	melee_damage_lower = 1
+	melee_damage_upper = 1
+	attacktext = "nibbled"
+	universal_speak = 1
+	universal_understand = 1
+
+	min_oxy = 1 //still require a /bit/ of air.
+	max_co2 = 0
+	unsuitable_atoms_damage = 1
+
+	supernatural = 1
+
+/mob/living/simple_animal/mouse/familiar/New()
+	..()
+	add_spell(new /spell/targeted/heal_target, "const_spell_ready")
+	add_spell(new /spell/targeted/heal_target/area, "const_spell_ready")
+	add_language("Ceti Basic")
```

Comment:
Talked to alb, apparently all familiars can.

Probably should use the language define though: `LANGUAGE_TCB`